docs/_documentations/troubleshooting.md

```diff
@@ -941,12 +941,16 @@ Check the Codewind output stream first when troubleshooting because it is partic
 
 If you report an issue, you will be asked to upload your logs.
 
-1. In VS Code, open **Help** > **Toggle Developer Tools**.
-2. Go to the **Console** tab.
-3. Enter *codewind.log* in the **Filter** box:
-<br>![Log location](images/cdt-vsc/logs-location.png)
+1. In VS Code, open the **Output** view.
+2. Select the Codewind output stream.
+3. The first line of the Codewind output stream is the location of the Codewind extension logs.
+<br><br>
```

Comment:
In a different file, I saw a line break that looked like this `<br/>` and just one on the line. Of course, if different things work for the same outcome, I understand. :)

Reply:
With and without the slash are equivalent